RELEASE.md

```diff
@@ -27,6 +27,7 @@ Steps to release a new version of the Jaeger Operator:
    ```sh
    git commit -sm "Preparing release v1.30.0"
    ```
+    Note that the OpenShift CI test job fails. This is expected and the release PR can be merged.
```

Comment:
@pavolloffay The CI job, builds the operand and operator bundle. The operand image is replaced in the bundle manifest using substitutions step like below in the CI job and then the bundle is built using it.

```
operator:
  bundles:
  - as: jaeger-bundle
    context_dir: .
    dockerfile_path: bundle.Dockerfile
  substitutions:
  - pullspec: quay.io/jaegertracing/jaeger-operator:1.49.0 <<-- Searches for this image pull spec in the bundle manifest.
    with: pipeline:jaeger-operator <<-- s operand image built in the CI job.
```
Refer https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles
  
  Current bundle step in the job config:
  https://github.com/openshift/release/blob/master/ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main.yaml#L43 

The release PR has the updated operand image  quay.io/jaegertracing/jaeger-operator:1.51.0 https://github.com/jaegertracing/jaeger-operator/pull/2365/files#diff-7b7a3ec017ac9ee661ba548ac43fe6fd698c741b21a09851971a33d618f0c963R410 but the CI job config tries to look for the operand image with 1.49.0 tag and replace it with the CI built operand image which doesn't work. 

To resolve this issue we have to update the operand tag in the CI job config after the release PR is merged. This is the current open PR to do so. https://github.com/openshift/release/pull/45835/files#diff-aba50e89c2b0f9274fa7a289c152946ffbc197736779c0772638d461d0298af4R44

To avoid running into this issue, I'm checking if a regex can be used to replace the operand image and not to hard-code the version tag in the job config.

Reply:
@pavolloffay I have added the regex for the image tags, so this will be resolved with https://github.com/openshift/release/pull/45835/files#diff-aba50e89c2b0f9274fa7a289c152946ffbc197736779c0772638d461d0298af4R44 We can remove that note.

Reply:
thanks for the clarification @IshwarKanse